src/app/zones/views/ZoneCreateView.vue

```diff
@@ -498,3 +498,8 @@ function toggleConfirmModal() {
   isConfirmModalVisible.value = !isConfirmModalVisible.value
 }
 </script>
+<style lang="scss" scoped>
+.radio-button-group > * + * {
+  margin-block-start: $kui-space-40;
+}
+</style>
```

Comment:
**Question**: This shouldn’t be necessary as this is available as a global style. Can you check again whether this is still necessary (could be related to the stylesheet ordering issue I fixed earlier perhaps)?

Reply:
I just noticed it’s actually removed from the global styles. Why?